<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>captureTool</class>
 <widget class="QWidget" name="captureTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(147, 147, 147);</string>
  </property>
  <widget class="QLabel" name="torqueOutputLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Torque Output: </string>
   </property>
  </widget>
  <widget class="QLabel" name="torqueNumber">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="lowTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="modelTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="hiLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectToolLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Capture Tool</string>
   </property>
  </widget>
  <widget class="QLabel" name="milwaukeeLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>531</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>mt1.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="captureButton_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lowLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Low: </string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="passLabel_2">
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>PASS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="failLabel_2">
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>FAIL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="hiTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
